controllers/src/main/scala/cromwell/pipeline/controller/utils/CustomPathMatcher.scala

```diff
@@ -0,0 +1,12 @@
+package cromwell.pipeline.controller.utils
+
+import akka.http.scaladsl.server
+import akka.http.scaladsl.server.PathMatchers.Segment
+import cromwell.pipeline.datastorage.dto
+
+import java.nio.file.{ Path, Paths }
+
+object CustomPathMatcher {
```

Comment:
```suggestion
object PathMatchers {
```
It's obvious that these path matchers are custom, otherwise we wouldn't have defined them at all :) So we can just make the name similar to predefined matchers

Reply:
Ok. I Agree ;) I`ll fix it.

Reply:
Done.